content/projects/github-api-consume/_index.md

```diff
@@ -50,3 +50,9 @@ Please be sure to follow the standard naming conventions for your language.
 ## Resources
 
 - [An introduction to curl using GitHub's API](https://gist.github.com/tazjel/8735770).
+
+### Instructions for reviewers
+- Ensure that the function works with any github User and repository name, meaning that the function should not only work with one User/repo name. 
+- Ensure that the function prints and not returns the output.
```

Comment:
- Ensure that the function prints and does not return the output

Reply:
Im pretty sure this should actually return the array/list. From the instructions 
> The function should output a list or array of pull requests on the repo such that the PRs were created, updated, merged or closed between the given two dates.

If we just need to print the data then there's no point to it being in any specific data structure. Remember when we get data from an api we want to do something with it, eg; it needs to be made available to some front end or it'll be written to file for some analysis. Either way the data needs to be **returned** rather than printed, otherwise nothing can be done with it.